vue/src/Tag/TagList.vue

```diff
@@ -135,7 +135,7 @@
               <span>{{ tag.updated_date_pretty }}</span>
             </td>
             <td
-              class="action"
+              :class="getActionClasses"
               v-show="hasWriteAccess"
```

Comment:
@snake14 Should we check `hasWriteAccess && hasWriteCapability` although it should be covered by `getActionClasses` check I beleieve

Reply:
@AltamashShaikh Correct. Adding a check here would be redundant. It's already handled in `getActionClasses` where it uses the `hasPublishCapability` function to check for write and custom templates.